tests/test_resources/negative-scenarios/test_pipeline_execution_fails_bad_syntax/Jenkinsfile

```diff
@@ -0,0 +1,14 @@
+pipeline {
```

Comment:
Out of curiosity where is the bad syntax?

Reply:
It's a declarative pipeline and the `step` tag is missing

Reply:
Maybe a clarifying comment can be added? (I'm not sure if it's possible within a declarative pipeline).